pallets/elections/src/lib.rs

```diff
@@ -88,23 +77,35 @@ pub mod pallet {
     #[pallet::without_storage_info]
     pub struct Pallet<T>(_);
 
+    /// List of possible validators, used during elections.
+    /// Can be modified via `change_members` call that requires sudo.
```

Comment:
afaik, usually you separate the brief description and the rest of docs with an empty `///` line

Reply:
done